File: Java/Film.java
// class cinema film
public class Film {
    private int id;
    private String judul;
    private String genre;

    // Constructor default
    public Film() {
        this.id = 0;
        this.judul = "";
        this.genre = "";
    }

    // Constructor berparameter
    public Film(int id, String judul, String genre) {
        this.id = id;
        this.judul = judul;
        this.genre = genre;
    }

    // Getter
    public int getId() { return id; }
    public String getJudul() { return judul; }
    public String getGenre() { return genre; }

    // Setter
    public void setId(int id) { this.id = id; }
    public void setJudul(String judul) { this.judul = judul; }
    public void setGenre(String genre) { this.genre = genre; }
}

File: PHP/film.php
<?php

// class cinema film
class Film {
    private int $id;
    private string $judul;
    private string $genre;

    // Constructor default / berparameter
    public function __construct(int $id = 0, string $judul = "", string $genre = "") {
        $this->id    = $id;
        $this->judul = $judul;
        $this->genre = $genre;
    }

    // Getter
    public function getId(): int      { return $this->id; }
    public function getJudul(): string { return $this->judul; }
    public function getGenre(): string { return $this->genre; }

    // Setter
    public function setId(int $id): void           { $this->id = $id; }
    public function setJudul(string $judul): void  { $this->judul = $judul; }
    public function setGenre(string $genre): void  { $this->genre = $genre; }
}